engine/render/renderer/platform/vulkan/Swapchain.h

```diff
@@ -96,7 +97,7 @@ class Swapchain
      * @param imageIndex - the index of the image being drawn
      * @return VkResult - the result of submitting the buffer
      */
-    Utils::Result SubmitCommandBuffers(const VkCommandBuffer* buffers, uint32_t* imageIndex);
+    Utils::Result SubmitCommandBuffers(CommandBuffer& commandBuffer, uint32_t imageIndex);
 
```

Comment:
Why is this non-const?

Reply:
We can safely make this `const`. Changed the function.